_specs/ecip-11355.md

```diff
@@ -0,0 +1,207 @@
+---
+ecip: 11355
```

Comment:
```suggestion
ecip: 1100
```

please also rename the file accordingly.

Reply:
https://github.com/ethereumclassic/ECIPs/pull/373/commits/b1984548ea34c1c692784b04ec5edadc26149258